### ISSUE TITLE
Bug: sort-named-exports cannot read properties of undefined (reading 'node')

### ISSUE THREAD

**Issue by Holiden** [azat-io/eslint-plugin-perfectionist#171]
### Describe the bug

The rule returns an error

```
TypeError: Cannot read properties of undefined (reading 'node')
Occurred while linting P:\...\packages\eslint\index.js:11
Rule: "perfectionist/sort-named-exports"

TypeError: Cannot read properties of undefined (reading 'node')
Occurred while linting P:\...\packages\eslint\index.js:11
Rule: "perfectionist/sort-named-exports"
    at makeFixes (file:///P:/.../node_modules/eslint-plugin-perfectionist/dist/utils/make-fixes.mjs:13:44)
    at Object.fix (file:///P:/.../node_modules/eslint-plugin-perfectionist/dist/rules/sort-named-exports.mjs:94:31)
    at normalizeFixes (P:\...\node_modules\eslint\lib\linter\report-translator.js:194:28)
    at P:\...\node_modules\eslint\lib\linter\report-translator.js:365:49
    at Object.report (P:\...\node_modules\eslint\lib\linter\linter.js:1047:41)
    at file:///P:/.../node_modules/eslint-plugin-perfectionist/dist/rules/sort-named-exports.mjs:87:21
    at pairwise (file:///P:/.../node_modules/eslint-plugin-perfectionist/dist/utils/pairwise.mjs:7:9)
    at ExportNamedDeclaration (file:///P:/.../node_modules/eslint-plugin-perfectionist/dist/rules/sort-named-exports.mjs:82:9)
    at ruleErrorHandler (P:\...\node_modules\eslint\lib\linter\linter.js:1076:28)
    at P:\...\node_modules\eslint\lib\linter\safe-emitter.js:45:58
```

Sample configuration for this rule

```
...
"perfectionist/sort-named-exports": [
  "error",
  {
    "group-kind": "values-first",
  },
],
```

### Code example

```js
const q = 1;
const w = 2;
const e = 3;

export { q, w, e };
```

### ESLint version

v8.57.0

### ESLint Plugin Perfectionist version

v2.11.0

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Hello!

Thank you for your issue. Unfortunately, I have not been able to reproduce the issue at this time.

Can you please suggest if you can create a repository and reproduce the issue there?
And maybe I need more information about your environment. Show me your package.json and ESLint config.

**Comment by Holiden** [comment]
Hi!

Thanks a lot for this plugin. 

[Here is](https://github.com/Holiden/perfectionist-sort-named-exports-error) a minimal repository with a repeat of this bug. There are only 2 rules in the config here. You can try to fix the file test.js. There will be an error in the console. If you delete the perfectionist/sort-named-exports rule and try to fix the file again, then you can see how the product object is sorted.

**Comment by azat-io** [comment]
Thank you for your issue!

The problem was fixed in a0208ace711a480cfa3de0091c5b701bba71cd4f and released in [v3.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.0.0).

If you'd like, you can support the release with a retweet:
https://x.com/azat_io_en/status/1815367279191761054
